test/xdpmp/rx.c

```diff
@@ -93,7 +131,8 @@ MpReceiveProcessBatch(
     // Perform action for each frame in the batch.
     //
     while (*StartIndex != FrameRing->ProducerIndex) {
-        Frame = XdpRingGetElement(FrameRing, (*StartIndex)++ & FrameRing->Mask);
+        UINT32 FrameRingIndex = (*StartIndex)++ & FrameRing->Mask;
```

Comment:
Not super important, but I like adding `const` to things we know won't/shouldn't change.

Reply:
Yeah, we should get in the habit of that.

Reply:
Will make a mental note but not fix this instance, if that's OK.